meetings/2022-12-22.md

```diff
@@ -53,6 +57,51 @@ URL: https://github.com/solid/notifications/pull/135
 
 >ACTION: eP to create issue discussing need for `channel` defined in the vocab or just in JSON-LD context.
 
+* RG: We didn't specify how does the request and response work when you want to modify the subscription
+* SC: Not in this PR, we still didn't reach consensus with this feature. We might need an issue for it.
+* RG: I think we have an issue for it.
+* eP: is there something problematic in 135 if we don't include updating now?
+* RG: No but it will have to do with the data model (eventually). I don't see a problem as another thing but it should be part of the next release.
+* SC: That feature would require both an update to the protocol and data model.
+* eP: RG, do you have a specific proposal? We can take care of this PR and then create issue to work together towards a PR for updating the subscription. Still need a clear approach.
+
+ACTION: RG to check existing issues or create an issue to track updating subscriptions.
+RG: https://github.com/solid/notifications/issues/103
+
+* SC: I tried to incorporate all of the comments. I think there are 2 or 3 which I still didn't get to.
+* ...: Pavlik suggested we could get rid of the terminology.
+* eP: In general, terminology makes sense if there is no dedicated section and a term used throughout the spec. On the contrary all the terms we have in terminology they have their own section. In that case those sections re normative and we can reuse. It could clean-up linking.
```

Comment:
```suggestion
* eP: In general, terminology makes sense if there is no dedicated section and a term is used throughout the spec. On the contrary, all the terms we have in terminology have their own section. In that case, those sections are normative, and we can reuse them. It could clean up linking.
```